app/presenters/tree_builder_vms_filter.rb

```diff
@@ -1,16 +1,12 @@
 class TreeBuilderVmsFilter < TreeBuilder
   def tree_init_options
     {
-      :open_all => true,
-      :leaf     => 'ManageIQ::Providers::InfraManager::Vm'
+      :open_all       => true,
+      :leaf           => 'ManageIQ::Providers::InfraManager::Vm',
+      :allow_reselect => true
     }
   end
 
-  def set_locals_for_render
-    locals = super
-    locals.merge!(:tree_id => "vms_filter_treebox", :tree_name => "vms_filter_tree", :allow_reselect => true)
```

Comment:
Where will these names come from now?

Both `vms_filter_tree` and `vms_instances_filter_tree` are still tested in code, but never passed to TreeBuilder*.new

Reply:
It's coming from the `vm_infra_controller` as a feature definition:
```ruby
  def features
    [
      ApplicationController::Feature.new_with_hash(
        :role  => "vandt_accord",
        :name  => :vandt,
        :title => _("VMs & Templates")
      ),

      ApplicationController::Feature.new_with_hash(
        :role  => "vms_filter_accord",
        :name  => :vms_filter,
        :title => _("VMs")
      ),

      ApplicationController::Feature.new_with_hash(
        :role  => "templates_filter_accord",
        :name  => :templates_filter,
        :title => _("Templates")
      ),
    ]
  end
```